extensions/http(s)/test/ace/http/request_test.exs

```diff
@@ -49,6 +49,15 @@ defmodule Ace.HTTP.RequestTest do
     assert upload.type == "text/plain"
   end
 
+  test "handles absoluteURI", %{port: port} do
+    request = """
+    GET http://www.raxx.com/ HTTP/1.1
```

Comment:
I'm not sure but I suspect that even though this is using an absolute url in the start-line there is still a requirement for a Host header. 

I started reading the spec but found it to be inconclusive https://tools.ietf.org/html/rfc7230#section-5.3.2